Subject: remove optional sc dep
From: Michael Tokarev <mjt@tls.msk.ru>
Forwarded: not-needed

sc (system call) crate isn't portable across architectures
debian is interested in. Disable its usage here for now.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54 +54 @@
-direct-syscall = ["sc"]
+#direct-syscall = ["sc"]
@@ -60,3 +60,3 @@
-[dependencies.sc]
-version = "0.2"
-optional = true
+#[dependencies.sc]
+#version = "0.2"
+#optional = true
